runtime/service_integ_test.go

```diff
@@ -2318,9 +2316,7 @@ func TestPauseResume_Isolated(t *testing.T) {
 	runTest := func(t *testing.T, request *proto.CreateVMRequest, state func(ctx context.Context, resp *proto.CreateVMResponse)) {
 		vmID := testNameToVMID(t.Name())
 
-		tempDir, err := ioutil.TempDir("", vmID)
-		require.NoError(t, err)
-		defer os.RemoveAll(tempDir)
+		tempDir := t.TempDir()
```

Comment:
Because of we no longer have `err` here, ResumeVM and StopVM use the err of the upper scope... That's why the tests are failing.

Reply:
Good catch :+1: